unicodetools/src/main/java/org/unicode/text/utility/Utility.java

```diff
@@ -1451,6 +1451,14 @@ public static String getMostRecentUnicodeDataFile(
             if (version != null && version.compareTo(currentVersion) < compValue) {
```

Comment:
FYI: Hopefully in the future we can completely stop looping over the versions in the searchPath...